src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs

```diff
@@ -40,6 +40,56 @@ public async Task InsertMutation(string dbQuery)
             SqlTestHelper.PerformTestEqualJsonStrings(expected, actual.ToString());
         }
 
+        /// <summary>
+        /// <code>Do: </code> Inserts new row and return all its columns
+        /// <code>Check: </code> A row is inserted in the table that has rows with default values as built_in methods.
+        /// it should insert it correctly with default values correctly handled by database.
+        /// current_date, current_timestamp, random_number, next_day have default value as built_in methods GETDATE(), NOW(), RAND(), DATEADD(), resp.
+        /// default_string_with_parenthesis has default value "()", default_function_string_with_parenthesis has default value "NOW()".
+        /// default_integer has default value 100, default_date_string has default value "1999-01-08 10:23:54".
+        /// Returned response would look like:
+        /// "createDefaultBuiltInFunction": {
+        ///   "current_date": "2023-12-15T16:24:48.267Z",
+        ///   "current_timestamp": "2023-12-15T16:24:48.267Z",
+        ///   "random_number": 0,
+        ///   "next_day": "2023-12-16T00:00:00.000Z",
+        ///   "default_string_with_paranthesis": "()",
+        ///   "default_function_string_with_paranthesis": "NOW()",
+        ///   "default_integer": 100,
+        ///   "default_date_string": "1999-01-08T10:23:54.000Z"
+        /// }
+        /// </summary>
+        public virtual async Task InsertMutationWithDefaultBuiltInFunctions()
```

Comment:
You should add one more (distinct) test which executes this mutation twice and saves both results. In this assertion section, validate that the `execution1.current_date != execution2.current_date` and all other function fields that should return different values for each request. 

This will validate that the error scenario Ani highlighted does not occur.

Reply:
Good suggestion - specifically for Pg/MySql